src/modules/storage/LedgerStorage.ts

```diff
@@ -569,13 +580,12 @@ export class LedgerStorage extends Storages
             });
         }
 
-        function update_spend_output (storage: LedgerStorage,
-            input: TxInput): Promise<void>
+        function update_spend_output (storage: LedgerStorage, input: TxInput): Promise<void>
         {
             return new Promise<void>((resolve, reject) =>
             {
                 storage.run(
-                    `UPDATE tx_outputs SET used = 1 WHERE utxo_key = ?`,
+                    `DELETE FROM utxos WHERE utxo_key = ?`,
```

Comment:
Nice! 
What should We do if there is an error in this part?

Reply:
For example, there is no utxo to erase, or there is throw error.

Reply:
That's a good point. I'm gonna have to test it out.

Reply:
When running a query using a non-existent UTXO, no error occurred as follows:
```
Execution finished without errors.
Result: query executed successfully. Took 0ms, 0 행이 영향받았습니다
At line 1:
DELETE FROM utxos WHERE utxo_key = x'4FAB6478E5283258DD749EDCB303E48F4192F199D742E14B348711F4BBB116B197E63429C6FA608621681E625BAF1B045A07ECF12F2E0B04C38BEE449F5EACFA'
```